src/main/java/duke/Todo.java

```diff
@@ -0,0 +1,67 @@
+package duke;
+
+import java.time.LocalDateTime;
+
+/**
+ * Task that has a description only.
+ */
+public class Todo extends Task {
+
+    /**
+     * Default constructor, isDone set to false.
+     * @param description of Todo
+     */
+    public Todo(String description) {
+        super(description);
+    }
+
+    public Todo(String description, Boolean isDone) {
+        super(description, isDone);
+    }
+
+    /**
+     * Constructor used to clone a Todo object
+     * @param todo
+     */
+    public Todo(Todo todo) {
+        super(todo);
+    }
+
+    @Override
+    public String toString() {
+        return "[T]" + super.toString();
+    }
+
+    /**
+     * Converts the database representation of Todo to a Todo Task
+     * @param dbTodo the string rep of Todo in the database
+     * @return Task the todo Task object
+     */
+    public static Todo db2Todo(String dbTodo) {
+        // T | 0 | Buy Bread
+        String[] params = dbTodo.split(" \\| ");
+        Boolean isDone = params[1].equals("1") ? true : false; // if "1", means isDone
+        String desc = params[2];
+        return new Todo(desc, isDone);
+    }
+
+    /**
+     * Converts the a Todo Task to the database representation of Todo.
+     * @param todoTask is the todo Task object
+     * @return Task the string rep of Todo in the database
+     */
+    public static String todo2Db(Todo todoTask) {
```

Comment:
Interesting and great way of shortening the function name while maintaining clarity!
Alone it might not be clear enough but with the comprehensive JavaDoc, it's great!👍